### DISCUSSION TITLE
Wont let me log in

### DISCUSSION THREAD

**Discussion by Boowhoowompwompl0s3r** [community/community#101903] [Mobile]
### Select Topic Area

Bug

### Body

Every time i try to log in it says im in but does not let me login n is taking forever basically not even opening the app after words smh

**Comment by JCCourt** [comment]
Hi @Boowhoowompwompl0s3r 

Welcome to the Github Community Forum!

There a several potential issues:

* Check Internet Connection:
  * Make sure you have a stable internet connection. Sometimes, connectivity issues can prevent the app from logging in.

* Clear App Cache:
  * Clearing the cache of the app may help resolve issues. Go to your device settings, find the GitHub app, and clear its cache.

* Update the App:
  * Ensure that you are using the latest version of the GitHub app. Outdated versions may have bugs that have been fixed in newer releases. Visit your app store to see if an update is available.

* Restart Your Device:
  * Sometimes, simply restarting your device can solve various app-related issues. Give it a try.

* Reinstall the App:
  * If the problem persists, uninstall the GitHub app, restart your device, and then reinstall the app from the app store.

* Check GitHub Status:
  * It's possible that GitHub itself is experiencing issues. Check the [GitHub Status](https://www.githubstatus.com/) page to see if there are any ongoing problems.

* Browser Login:
  * If the issue is specific to the app, try logging in through a web browser on your device to see if the problem persists. This can help determine if the issue is with your account or the app itself.

* Contact GitHub Support:
  * If none of the above steps resolves the issue, consider reaching out to GitHub support for assistance. You can find support options on the GitHub website or within the app itself.

Ask if you need anymore help 😊

-JCCourt- 

Mark helpful posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great content!

**Comment by Basiron** [comment]
![Screenshot_2024-01-25-12-45-38-992_com garena game codm](https://github.com/community/community/assets/156036496/ab9a8217-71be-45f2-9d4f-5c6763c64099)
